### DISCUSSION TITLE
v0.16.3

### DISCUSSION THREAD

**Discussion by erogol** [coqui-ai/TTS#2869] [Versions & Releases]
## What's Changed
* Update Studio API for XTTS by @erogol in https://github.com/coqui-ai/TTS/pull/2861
* Denote human voices in README.md by @michaelnew in https://github.com/coqui-ai/TTS/pull/2851

## New Contributors
* @michaelnew made their first contribution in https://github.com/coqui-ai/TTS/pull/2851

**Full Changelog**: https://github.com/coqui-ai/TTS/compare/v0.16.2...v0.16.3

<hr /><em>This discussion was created from the release <a href='https://github.com/coqui-ai/TTS/releases/tag/v0.16.3'>v0.16.3</a>.</em>

**Comment by phillip1335** [comment]
I've downloaded the zip file and extracted it into its own directory. How do I run it?

**Reply by erogol** [reply]
👁️ https://tts.readthedocs.io/en/latest/
